src/google/protobuf/compiler/parser_unittest.cc

```diff
@@ -2069,13 +2069,107 @@ TEST_F(ParserValidationErrorTest, Proto3Default) {
 
 TEST_F(ParserValidationErrorTest, Proto3JsonConflictError) {
   ExpectHasValidationErrors(
-      "syntax = 'proto3';\n"
-      "message TestMessage {\n"
-      "  uint32 foo = 1;\n"
-      "  uint32 Foo = 2;\n"
-      "}\n",
-      "3:9: The JSON camel-case name of field \"Foo\" conflicts with field "
-      "\"foo\". This is not allowed in proto3.\n");
+      R"pb(
```

Comment:
Please don't use R"pb()pb" for these.  We format those literals as *text protos*, not proto files and this is causing test failures due to reformatting.